src/components/SummaryCard/SummaryCard.stories.js

```diff
@@ -3,120 +3,183 @@
-              />
-              <SummaryCardAction
-                iconDescription="Icon description"
-                renderIcon={Folder20}
-                hasIconOnly
-                onClick={action('onClick')}
-                tooltipPosition="bottom"
-                tooltipAlignment="center"
-              />
-              <SummaryCardAction
-                iconDescription="Icon description"
-                renderIcon={Folder20}
-                hasIconOnly
-                onClick={action('onClick')}
-                tooltipPosition="bottom"
-                tooltipAlignment="center"
-              />
-            </SummaryCardFooter>
-          </SummaryCard>
-        </div>
+          </SummaryCardFooter>
+        </SummaryCard>
       </div>
-    </div>
+      <div
+        className={`${prefix}--col-sm-4 ${prefix}--col-md-4 ${prefix}--col-lg-4`}
+      >
+        <SummaryCard>
+          <SummaryCardHeader
+            title="Summary card with no footer"
+            status={
+              <Tooltip showIcon iconDescription="Status">
+                Tooltip content
+              </Tooltip>
+            }
+          />
+          <SummaryCardBody>{lorem.repeat(5)}</SummaryCardBody>
+        </SummaryCard>
+      </div>
+      <div
+        className={`${prefix}--col-sm-4 ${prefix}--col-md-4 ${prefix}--col-lg-4`}
+      >
+        <SummaryCard>
+          <SummaryCardHeader title="Summary card with no status" />
+          <SummaryCardBody>{lorem.repeat(3)}</SummaryCardBody>
+          <SummaryCardFooter>
+            <SummaryCardAction loading expandedContent={lorem.repeat(5)}>
+              Button label
+            </SummaryCardAction>
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+            <SummaryCardAction
+              iconDescription="Icon description"
+              renderIcon={Folder20}
+              hasIconOnly
+              onClick={action('onClick')}
+              tooltipPosition="bottom"
+              tooltipAlignment="center"
+            />
+          </SummaryCardFooter>
+        </SummaryCard>
+      </div>
+    </Fragment>
+  ))
+  .add('Multiselect', () => (
+    <SummaryCardContainer
+      render={({
+        getBatchActionProps,
+        getSelectionProps,
+        selectedSummaryCards,
+        summaryCards,
+      }) => (
+        <Fragment>
+          <SummaryCardBatchActions {...getBatchActionProps()}>
+            <SummaryCardBatchAction
```

Comment:
I realize this component is based on `TableBatchAction` -- in that context, I was wondering how someone would be able to customize the icon used here? It looks like the "trash"/delete icon is used in the mockups, but this defaults to a filled plus icon?

Reply:
Good catch! It resorts to the default icon that Carbon sets for the `BatchAction`, but for the purposes of this story, I'll use the trash icon.